sites/partners/package.json

```diff
@@ -53,6 +53,7 @@
     "@babel/preset-env": "^7.11.5",
     "@cypress/code-coverage": "^3.9.12",
     "@cypress/webpack-preprocessor": "^5.11.1",
+    "@netlify/plugin-nextjs": "4.30.4",
```

Comment:
This was needed to be added because Netlify would always deploy with version 3.*. This is despite the netlify docs saying it will grab the latest. Ideally this shouldn't be here, but in the meantime hopefully this works until we are off netlify or the netlify issue is resolved